docs/source/models.rst

```diff
@@ -141,19 +140,19 @@ Predictors that localize and identify text elements in images
 +--------------------------------------------------------------+--------------------------------------+--------------------------------------+
 |                                                              |                  FUNSD               |                  CORD                |
 +=============================+=================+==============+============+===============+=========+============+===============+=========+
-|      **Architecture**       | **Input shape** | **# params** | **Recall** | **Precision** | **FPS** | **Recall** | **Precision** | **FPS** |
+| **Architecture**            | **Input shape** | **# params** | **Recall** | **Precision** | **FPS** | **Recall** | **Precision** | **FPS** |
 +-----------------------------+-----------------+--------------+------------+---------------+---------+------------+---------------+---------+
-| db_resnet50 + crnn_vgg16_bn | (1024, 1024, 3) |              |   0.648    |     0.703     |  0.85   |    0.677   |     0.784     |   1.6   |
+| db_resnet50 + crnn_vgg16_bn | (1024, 1024, 3) |              | 70.08      | 74.77         | 0.85    | 82.19      | 79.67         | 1.6     |
```

Comment:
maybe we can put in **bold** CORD precision for our model :)

Reply:
Good idea, I'll fix that!